tests/test-recorder.c

```diff
@@ -475,7 +475,7 @@ test_recorder_split_bytes (TestFixture * fixture, gconstpointer unused)
 
     if (filenames->next) {
       g_assert_cmpint (labs (stat_buf.st_size - FILE_SEGMENT_LEN_BYTES), <=,
-          FILE_SEGMENT_LEN_BYTES / 5);
+          FILE_SEGMENT_LEN_BYTES / 5 * 1.1);
```

Comment:
Perhaps make more obvious where did this `1.1` number come from?

Reply:
I see you point, would something like this be enough?

```
// Ratio to apply in file segment length check to ensure proper segment size
// and distribution
#define FILE_SEGMENT_LEN_BYTES_RATIO_LIMIT 1.1
```

Reply:
Hm, can we express the allowed size difference as a percentage of `FILE_SEGMENT_LEN_BYTES`? For example `FILE_SEGMENT_LEN_BYTES * 0.25`, meaning the resulting file should differ at most 25% from the expected size? (I thought that number up. Check experimentally which value works!)

Reply:
Sure, I just thought that somehow the `/ 5` had some meaning and tried to keep it. I do agree that `* 0.25` looks clearer.

Experiments show that `0.25` is enough so I will push that change.

Reply:
Can you add a comment that the ratio is only for testing purposes? Allowing 25% error may be a problem in actual use.

Reply:
Sure.

Reply:
After some additional testing with `azure` this change seems not to be needed, only in my PC this error may arise with low probability, so I'm dropping it.